src/entities/organization.ts

```diff
@@ -43,7 +43,6 @@ export class Organization extends BaseEntity {
   @Field(type => [Token], { nullable: true })
   @ManyToMany(type => Token, token => token.organizations, {
     nullable: true,
-    eager: true,
```

Comment:
Let me know if this is required, Mohammad, I needed it for the admin bro on the token side, but typeorm wont let you have it in both side.

Reply:
No carlos, it's not required, currently we dont need `eager` in here